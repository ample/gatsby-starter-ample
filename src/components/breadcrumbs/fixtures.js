export default {
  default: {
    links: [
      { icon: "home", label: "Home", url: "#" },
      { label: "Lorem", url: "#" },
      { label: "ipsum", url: "#" },
      { label: "dolor", url: "#" },
      { label: "sit", url: "#" },
      { label: "amet", url: "#" }
    ]
  }
}
