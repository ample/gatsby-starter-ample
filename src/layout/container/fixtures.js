import { fixtures as formFixtures } from "@src/components/form"

export default {
  empty: {
    className: null,
    components: [],
    config: {}
  },
  default: {
    className: null,
    components: [
      {
        template: "component-button",
        children: "Hello World",
        to: "/"
      }
    ],
    config: {}
  },
  form: {
    className: null,
    components: [{ template: "component-form", form: formFixtures.default }],
    config: {}
  }
}
