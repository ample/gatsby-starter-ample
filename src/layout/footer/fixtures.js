export default {
  default: {
    copyright: "2020, All Rights Reserved",
    menus: [
      {
        heading: "Quisque",
        links: [
          { url: "#", label: "Vestibulum vitae" },
          { url: "#", label: "Pellentesque tempor venenatis" },
          { url: "#", label: "Fusce molestie" },
          { url: "#", label: "Etiam in lacinia dolor" }
        ]
      },
      {
        heading: "Vestibulum",
        links: [
          { url: "#", label: "Fusce molestie" },
          { url: "#", label: "Pellentesque venenatis" },
          { url: "#", label: "Vestibulum vitae" },
          { url: "#", label: "lacinia dolor" }
        ]
      }
    ],
    policy_links: [
      { url: "#", label: "Privacy Policy" },
      { url: "#", label: "Terms of Service" }
    ],
    social_links: [
      { url: "#", icon: "twitter" },
      { url: "#", icon: "facebook" }
    ]
  }
}
