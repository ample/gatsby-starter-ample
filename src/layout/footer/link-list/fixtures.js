export default {
  default: {
    label: "Quisque",
    links: [
      { url: "#", label: "Vestibulum vitae" },
      { url: "#", label: "Pellentesque tempor venenatis" },
      { url: "#", label: "Fusce molestie" },
      { url: "#", label: "Etiam in lacinia dolor" }
    ]
  }
}
