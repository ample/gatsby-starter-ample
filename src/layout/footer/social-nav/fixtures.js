export default {
  default: {
    links: [
      { url: "#", icon: "twitter" },
      { url: "#", icon: "facebook" }
    ]
  }
}
