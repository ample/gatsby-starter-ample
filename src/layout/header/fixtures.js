export const header_test_data = {
  main_navigation: [
    { url: "#", label: "Reprehenderit" },
    { url: "#", label: "Officia sint" },
    { url: "#", label: "Veniam " },
    { url: "#", label: "Lacinia dolor" },
    { url: "#", label: "Consequat " },
    { url: "#", label: "Voluptate nostrud" }
  ],
  top_navigation: [
    { url: "#", label: "Duis" },
    { url: "#", label: "Cupidatat" },
    { url: "#", label: "Ullamco" },
    { url: "#", label: "Labore" },
    { url: "#", label: "Consectetur" },
    { url: "#", label: "Pariatur" }
  ]
}
