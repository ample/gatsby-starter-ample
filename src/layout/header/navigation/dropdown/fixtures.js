import React from "react"

// import fixtures from "./menu/fixtures.js"
// import { DropdownMenu, DropdownTrigger } from "."

export default {
  simple_dropdown: {
    children: (
      <>
        {/* <DropdownTrigger>Hover over me</DropdownTrigger> */}
        {/* <DropdownTrigger>Hover over me</DropdownTrigger>
        <DropdownMenu {...fixtures.simple_menu} /> */}
      </>
    )
  }
}
