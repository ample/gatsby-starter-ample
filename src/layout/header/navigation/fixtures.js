export default {
  default: {
    main_navigation: [
      { url: "#", label: "Reprehenderit" },
      { url: "#", label: "Officia sint" },
      { url: "#", label: "Veniam" },
      { url: "#", label: "Consequat" },
      {
        url: "#",
        label: "Suspendisse semper",
        children: [
          { url: "/", label: "Vestibulum vitae" },
          { url: "#", label: "Pellentesque tempor venenatis" },
          { url: "#", label: "Fusce molestie" },
          { url: "#", label: "Etiam in lacinia dolor" }
        ]
      },
      { url: "#", label: "Quisque interdum", button: true, className: "button" }
    ],
    top_navigation: [
      { url: "#", label: "Duis" },
      { url: "#", label: "Cupidatat" },
      { url: "#", label: "Ullamco" },
      { url: "#", label: "Labore" },
      { url: "#", label: "Consectetur" },
      { url: "#", label: "Pariatur" }
    ]
  }
}
