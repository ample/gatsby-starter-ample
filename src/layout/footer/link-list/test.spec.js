import React from "react"
import renderer from "react-test-renderer"

import LinkList from "."

import fixtures from "./fixtures"

describe("LinkList", () => {
  it("renders link list correctly", () => {
    const tree = renderer.create(<LinkList {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
