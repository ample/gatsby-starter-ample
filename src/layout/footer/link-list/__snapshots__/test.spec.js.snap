// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkList renders link list correctly 1`] = `
<ul
  className="link_list"
>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Vestibulum vitae
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Pellentesque tempor venenatis
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Fusce molestie
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Etiam in lacinia dolor
    </a>
  </li>
</ul>
`;
